docs/learn/learn-comparison-ethereum-2.md

```diff
@@ -106,83 +104,79 @@ Ethereum operates as a single, homogeneous chain. Each Ethereum node is divided
 
 ### Forks, Upgrades, and Governance
 
-Ethereum governance is done off-chain, where various stakeholders come to a consensus through some
+Ethereum's governance is done off-chain, where various stakeholders come to a consensus through some
 medium other than the protocol itself. Upgrades on Ethereum will follow the standard hard-fork
-procedure, requiring validators to upgrade their nodes to implement protocol changes.
+procedure, coordinating both the community and validators to upgrade their nodes to implement
+protocol changes.
 
 {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} uses on-chain
-[governance](./learn-polkadot-opengov.md) with a multicameral system. There are several avenues to
-issue proposals, and all proposals ultimately pass through a public referendum, where the majority
-of tokens can always control the outcome.
-{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} uses
-[adaptive quorum biasing](./learn-governance.md#adaptive-quorum-biasing) to set the passing
-threshold for low-turnout referenda. Referenda can cover various topics, including fund allocation
-from an on-chain [Treasury](./learn-treasury.md) or modifying the underlying runtime code of the
-chain. Decisions get enacted on-chain and are binding and autonomous.
-
-{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} can enact chain upgrades and
-successful proposals using the Wasm meta-protocol without a hard fork. Anything within the state
-transition function, the transaction queue, or off-chain workers can be upgraded without forking the
-chain.
-
-### Consensus and Finalization
-
-Ethereum and {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} use hybrid consensus
-models where block production and finality have their protocols. The finality protocols - Casper FFG
-for Ethereum and [GRANDPA](./learn-consensus.md#finality-gadget-grandpa) for
-{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} - are both GHOST-based and can both
-finalize batches of blocks in one round. For block production, both protocols use slot-based
-protocols that randomly assign validators to a slot and provide a fork choice rule for unfinalized
-blocks - RandDAO/LMD for Ethereum and [BABE](./learn-consensus.md#badass-babe-sassafras) for
-{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}.
+[governance, called OpenGov](./learn-polkadot-opengov.md), to facilitate upgrades to the Polkadot
+runtime. These upgrades are voted on via the stakeholders of Polkadot, where if successful, the
+upgrade is enacted automatically in the blocks to come. The only time Polkadot validator operators
+upgrade their nodes is when the client itself gets updated.
+
+Because of this mechanism, {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} can enact
+chain upgrades and successful proposals using the Wasm meta-protocol _without_ a hard fork. Anything
```

Comment:
I think Bill described this really well in its ETH Denver speech. The runtime is stored on chain -> this allows forkless upgrades because it is not stored in the node client. Maybe we can expand a little with a few more sentences.

Reply:
fixed - let me know how it is!